test/shared/config/SharedAppConfigSpec.scala

```diff
@@ -89,6 +89,16 @@ class SharedAppConfigSpec extends UnitSpec {
         expectedTysIfsEnvHeaders
       )
     }
+
+    "return the apiDocumentationUrl" when {
+      "it is not specified" in {
+        simpleAppConfig.apiDocumentationUrl shouldBe s"https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/${simpleAppConfig.appName}"
```

Comment:
This test confused us slightly, at first glance we thought the test cases were named the wrong way round. Possibly would be more clear if you used 

```suggestion
       val changedAppConfig = appConfig("", None)
        changedAppConfig.apiDocumentationUrl shouldBe s"https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/${simpleAppConfig.appName}"
```